extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Layout} from '../../../src/layout';
+// import * as st from '../../../src/style';
+import {addListener} from './instrumentation';
+import {AnalyticsConfig} from './vendors';
+import {platformVariablesFor, platformVariablesExpressionFor} from
+  './platform-variables';
+
+export class AmpAnalytics extends AMP.BaseElement {
+
+  constructor(element) {
+    super(element);
+    this.config_ = undefined;
+    this.type_ = undefined;
+    this.requests_ = {};
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return layout == Layout.FIXED;
+  }
+
+  /** @override */
+  firstAttachedCallback() {
+    let config = this.element.textContent;
+    // TODO: Add try catch and other error conditions below.
+    config = JSON.parse(config);
+    this.config_ = config;
+
+    const type = this.element.getAttribute('type');
+    if (type && AnalyticsConfig[type]) {
+      this.type_ = type;
+
+      // Handle Optout.
+      if (AnalyticsConfig[type]['optout'] &&
```

Comment:
Alternatively, you can keep quotes but then you have to add quotes in the AnalyticsConfig properties. Basically it should be the same here and there.